portfolio/src/main/webapp/script.js

```diff
@@ -36,6 +36,34 @@ async function getCommentUsingAsyncAwait(){
     commentContainer.innerText = comments;
 }
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
```

Comment:
Did you have to make any other changes than the script changes? a new pom dependency or script imports etc?